doc/classes/Image.xml

```diff
@@ -238,51 +253,56 @@
 			<argument index="0" name="rect" type="Rect2">
 			</argument>
 			<description>
-				Return a new [code]Image[/code] that is a copy of "area" in this [code]Image[/code].
+				Returns a new image that is a copy of the image's area specified with [code]rect[/code].
 			</description>
 		</method>
 		<method name="get_size" qualifiers="const">
 			<return type="Vector2">
 			</return>
 			<description>
+				Returns a [code]Vector2[/code] that is the size of the image.
```

Comment:
- The return type is already indicated (note you didn't say "Returns an [int] of the image's width.")
- Use possessive over "of the"

Returns the image's size (width and height).